scan/src/pages/ValidatorHomePage.re

```diff
@@ -148,12 +157,18 @@ let make = () => {
     </div>
     <div className=Styles.highlight>
       <Row>
-        <Col size=0.7> <InfoHL info={InfoHL.Fraction(8, 20, false)} header="VALIDATORS" /> </Col>
+        <Col size=0.7>
+          <InfoHL
+            info={InfoHL.Fraction(bondedValidatorCount, allValidatorCount, false)}
+            header="VALIDATORS"
+          />
+        </Col>
+        // MOCK for now
         <Col size=1.1>
-          <InfoHL info={InfoHL.Fraction(5352500, 10849023, true)} header="BONDED TOKENS" />
+          <InfoHL info={InfoHL.Fraction(400, 400, true)} header="BONDED TOKENS" />
```

Comment:
Make 400 come from sum of all validators
Another 400 can you get from globalcontext create new variable to store and mock it